scripts/triggers-end-to-end-tests/triggers/index.js

```diff
@@ -70,6 +95,18 @@ exports.writeToAuth = functions.https.onRequest(async (req, res) => {
   res.json({ created: "ok" });
 });
 
+exports.createUserFromAuth = functions.https.onRequest(async (req, res) => {
+  await createUserWithEmailAndPassword(auth, "email@gmail.com", "mypassword");
+
+  res.json({ created: "ok" });
+});
+
+exports.signInUserFromAuth = functions.https.onRequest(async (req, res) => {
+  await signInWithEmailAndPassword(auth, "email@gmail.com", "mypassword");
```

Comment:
In theory, we don't need to init and use the JS SDK - instead we can call the REST API directly (https://firebase.google.com/docs/reference/rest/auth/#section-sign-in-email-password).

This will be more bloated, but could be less weird (weird to me since we are using JS SDK on server-side)[

Reply:
hmm, yeah that makes sense, lemme try and put something together that use it

Reply:
Well if we can't get rid of using admin sdk entirely... then I'm fine leaving this as is!

Reply:
Yeah, we'd be able to get rid of the JS SDK not the admin SDK